Makefile

```diff
@@ -35,29 +66,36 @@ APPVERSION   = "$(APPVERSION_M).$(APPVERSION_N).$(APPVERSION_P)"
 VARIANT_PARAM = COIN
 VARIANT_VALUES = BOL
 
-CURVE_APP_LOAD_PARAMS = secp256k1
-PATH_APP_LOAD_PARAMS = "44'/1'"   # purpose=coin(44) / coin_type=Testnet(1)
+# Enabling DEBUG flag will enable PRINTF and disable optimizations
+#DEBUG = 1
 
-ICON_NANOS = icons/nanos_app_boilerplate.gif
-ICON_NANOX = icons/nanox_app_boilerplate.gif
-ICON_NANOSP = icons/nanox_app_boilerplate.gif
-ICON_STAX = icons/stax_app_boilerplate_32px.gif
+########################################
+#     Application custom permissions   #
+########################################
+# See SDK include/appflags.h for the purpose of each permission
+#HAVE_APPLICATION_FLAG_DERIVE_MASTER = 1
+#HAVE_APPLICATION_FLAG_GLOBAL_PIN = 1
+#HAVE_APPLICATION_FLAG_BOLOS_SETTINGS = 1
+#HAVE_APPLICATION_FLAG_LIBRARY = 1
 
+########################################
+# Application communication interfaces #
+########################################
 ENABLE_BLUETOOTH = 1
+#ENABLE_NFC = 1
 
-# Enabling DEBUG flag will enable PRINTF and disable optimizations
-DEBUG ?= 0
-
-ENABLE_NBGL_QRCODE   = 1
-ENABLE_NBGL_KEYBOARD = 0
-ENABLE_NBGL_KEYPAD   = 0
+########################################
+#         NBGL custom features         #
+########################################
+ENABLE_NBGL_QRCODE = 1
+#ENABLE_NBGL_KEYBOARD = 1
+#ENABLE_NBGL_KEYPAD = 1
 
-APP_SOURCE_PATH += src
-
-#CUSTOM_APP_FLAGS = 0x000 # Set custom flags here, see appflags.h in SDK for flags definition
+########################################
+#          Features disablers          #
+########################################
 #DISABLE_STANDARD_APP_FILES = 1 
```

Comment:
Maybe we could add a comment stating it is relevant to the standard Makefile in the SDK